app/main.py

```diff
@@ -1 +1,51 @@
-# write your code here
+from __future__ import annotations
+
+
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name: str = name
+        self.skills: list[str] = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        self.skills: list[str] = ["JavaScript", "HTML", "CSS"]
```

Comment:
https://github.com/sashokpilipok7/py-software-engineers/blob/develop/checklist.md
Use extend to modify already created list
Fix in all places